manim/utils/images.py

```diff
@@ -32,3 +39,19 @@ def invert_image(image: np.array) -> Image:
     arr = np.array(image)
     arr = (255 * np.ones(arr.shape)).astype(arr.dtype) - arr
     return Image.fromarray(arr)
+
+
+def change_to_rgba_array(image):
+    """Converts an RGB array into RGBA with the alpha value opacity maxed."""
+    pa = image
+    if len(pa.shape) == 2:
+        pa = pa.reshape(list(pa.shape) + [1])
+    if pa.shape[2] == 1:
+        pa = pa.repeat(3, axis=2)
+    if pa.shape[2] == 3:
+        alphas = 255 * np.ones(
+            list(pa.shape[:2]) + [1],
+            dtype="uint8",
+        )
+        pa = np.append(pa, alphas, axis=2)
+    return pa
```

Comment:
Is this the same as https://github.com/ManimCommunity/manim/blob/9bae654ae688d27b04c6663f225072285cbaa734/manim/mobject/types/image_mobject.py#L191-L204 right? In which I would like to deprecate that one let `ImageMobject` use this one instead.